geode-redis/src/main/java/org/apache/geode/redis/internal/KeyRegistrar.java

```diff
@@ -201,11 +172,7 @@ private void throwDataTypeException(ByteArrayWrapper key, RedisData data) {
   }
 
   private void throwDataTypeException(ByteArrayWrapper key, RedisDataType dataType) {
-    if (RedisDataType.REDIS_PROTECTED.equals(dataType)) {
-      throw new RedisDataTypeMismatchException("The key name \"" + key + "\" is protected");
-    } else {
-      throw new RedisDataTypeMismatchException(
-          RedisConstants.ERROR_WRONG_TYPE);
-    }
+    throw new RedisDataTypeMismatchException(
```

Comment:
Doesn't look like the arguments are needed anymore. In fact, the whole function may be eliminated...

Reply:
done